content/en/news/releases/1.12.x/announcing-1.12/change-notes/index.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Change Notes
```

Comment:
Note that we probably want Release 1.12 in the title. See the images I appended earlier.

Reply:
I also notice a slight discrepancy, but it's on older releases, that the title says Change Notes and the description says Release Notes.